File: java/fundamentos/mostrar_resumen_de_contactos.java
package fundamentos;
public class mostrar_resumen_de_contactos {
    public static void main(String[] args) {
        int cantidadContactos = 3;
        String nombre1 = "Ana Gómez";
        String nombre2 = "Luis Pérez";
        String nombre3 = "Sofía Martínez";
        System.out.println("=== RESUMEN DE CONTACTOS ===");
        System.out.println("Cantidad total de contactos: " + cantidadContactos);
        System.out.println("Contactos registrados:");
        System.out.println("- " + nombre1);
        System.out.println("- " + nombre2);
        System.out.println("- " + nombre3);
    }
}

File: visual_basic/fundamentos_del_lenguaje/mostrar_resumen_de_contactos/mostrar_resumen_de_contactos.vb
Imports System
Module mostrar_resumen_de_contactos
    Sub Main(args As String())
        Dim nombre1 As String = "Juan Pérez"
        Dim nombre2 As String = "María López"
        Dim nombre3 As String = "Carlos Gómez"
        Dim totalContactos As Integer = 3
        Console.WriteLine("=== RESUMEN DE CONTACTOS ===")
        Console.WriteLine("Total de contactos: " & totalContactos)
        Console.WriteLine("Contactos:")
        Console.WriteLine("- " & nombre1)
        Console.WriteLine("- " & nombre2)
        Console.WriteLine("- " & nombre3)
        Console.ReadLine()
    End Sub
End Module